README.md

```diff
@@ -108,6 +108,58 @@ kubectl -n openobserve port-forward svc/openobserve 5080:5080
 ```
 ![Openobserve logs](docs/assets/openobserve-logs.png)
 
+### Sending logs to logging-operator (example)
+
+Install dependencies (cert-manager and opentelemetry-operator):
+
+```sh
+helm upgrade \
```

Comment:
I would recommend replacing this here (and everywhere, like I did [here](https://github.com/kube-logging/telemetry-controller/pull/65/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R144)) with the oneliner to be consistent with the opentelemetry operator install

Reply:
I chose for the `helm` one just because it would wait until the deployment was ready.
Anyhow I will change it accordingly.